### ISSUE TITLE
Player Pages only show results from the recent results.

### ISSUE THREAD

**Issue by AdmiralGT** [AdmiralGT/ttennis_ladder#49]
The recent results for a player page should display the previous N results no matter when they occured. If you look at an inactive player, they have no recent results.

**Comment by derek-cheong** [comment]
Should "recent results" actually be called "previous results"?

**Comment by AdmiralGT** [comment]
I don't think so. Should we ever move to a database environment then it would actually be the recent results. Also by definition all results are "previous" unless you've gained some sort of almanac from a time travelling pair.
